cmd/dep/init.go

```diff
@@ -36,6 +33,10 @@ versions available from the upstream source per the following algorithm:
  - Default branch(es) (sorted lexicographically)
  - Non-semver tags (sorted lexicographically)
 
+If configuration files for other dependency management tools are found, they
+are used to pre-populate the manifest and lock. Specify -skip-tools to disable
+this behavior.
+
 A Gopkg.toml file will be written with inferred version constraints for all
```

Comment:
This text needs to be updated, as well. (Can be a little sloppy about it, given that it's going to change again once we add the `-gopath` flag.)